process/smartContract/process.go

```diff
@@ -2608,36 +2611,48 @@ func (sc *scProcessor) processSCOutputAccounts(
 	return createdAsyncCallback, scResults, nil
 }
 
+func addCodeHashInLogEntryTopics(entry *vmcommon.LogEntry, codeHash []byte) {
+	if entry == nil {
+		return
+	}
+	if string(entry.Identifier) != core.SCUpgradeIdentifier && string(entry.Identifier) != core.SCDeployIdentifier {
```

Comment:
a little bit confusing. better do something like this:
```
isExpectedIdentifier := string(entry.Identifier) == core.SCUpgradeIdentifier || string(entry.Identifier) == core.SCDeployIdentifier
if !isExpectedIdentifier {
    return
}
```

Reply:
done